pkg/apis/submariner/v1alpha1/lighthouse_types.go

```diff
@@ -0,0 +1,66 @@
+package v1alpha1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// EDIT THIS FILE!  THIS IS SCAFFOLDING FOR YOU TO OWN!
+// NOTE: json tags are required.  Any new fields you add must have json tags for the fields to be serialized.
+
+// LighthouseSpec defines the desired state of Lighthouse
+// +k8s:openapi-gen=true
+type LighthouseSpec struct {
```

Comment:
What do you think using "SubmarinerServiceDiscovery" as we talked around lately?

That way it'd be less confusing about the details...
but may be... with the birth of subctl that doesn't really matter anymore?

Reply:
Was wondering about it too. Using SubmarinerServiceDiscovery for CRD name will make it too mouthful, more so since this will be prefix for most APIs. SubmarinerSD could be other option.